### ISSUE TITLE
Failed to compile 

### ISSUE THREAD

**Issue by Norfeldt** [pgengoux/react-native-huawei-protected-apps#1]
I have previous had good luck with this package, but now that I try to combine it with `react-native-firebase` it fails to compile to a real android device.

The `react-native-firebase` requires some fiddling within the native android code - so the might be the reason why it fails - I'm just not able to fix it and ask for a little help.

I hooked up `react-native-firebase` first (native code fiddling) and then I ran the `react-native link`.

```
../node_modules/react-native-huawei-protected-apps/android/src/main/java/com/github/pgengoux/huaweiprotectedapps/HuaweiProtectedAppsPackage.java:25: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-huawei-protected-apps:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-huawei-protected-apps:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

**Comment by skymax406** [comment]
Is just because since react-native@0.47 @Override is deprecated, remove 

```
    @Override
    public List<Class<? extends JavaScriptModule>> createJSModules() {
        return Collections.emptyList();
    }
```

from ../node_modules/react-native-huawei-protected-apps/android/src/main/java/com/github/pgengoux/huaweiprotectedapps/HuaweiProtectedAppsPackage.java

and it work :p

**Comment by pgengoux** [comment]
Fixed using pull request #2 from @elieteyssedou 
Sorry for the delay!
